src/main/kotlin/org/jitsi/nlj/transform/node/incoming/BitrateCalculator.kt

```diff
@@ -74,18 +76,42 @@ class VideoBitrateCalculator(
     override fun trace(f: () -> Unit) = f.invoke()
 }
 
-open class BitrateCalculator(name: String = "Bitrate calculator") : ObserverNode(name) {
+open class BitrateCalculator(
+    name: String = "Bitrate calculator",
+    /**
+     * At what threshold the stream is considered active.
+     */
+    private val packetRateThreshold: Int = 5
+) : ObserverNode(name) {
     private val bitrateStatistics = RateStatistics(5000, 8000f)
     private val packetRateStatistics = RateStatistics(5000, 1000f)
     val bitrate: Bandwidth
         get() = bitrateStatistics.rate.bps
     val packetRatePps: Long
         get() = packetRateStatistics.rate
 
+    // Keep track of whether the stream is active (has packets at at least [packetRateThreshold]
+    var active = true
```

Comment:
Should this be public?

Reply:
Yes, it's used to read whether the stream is active: https://github.com/jitsi/jitsi-videobridge/pull/1344/files#diff-583ae4d8b9cc2640c234d0c0ede1dd1dR199

Reply:
`private set` then?